administrator-guides/federation/README.md

```diff
@@ -0,0 +1,88 @@
+# Federation
+
+## Introduction
+
+Federation allows communication with other Rocket.Chat servers.
+
+When you open Federation's config screen, you will see this:
+
+![Initial Federation Screen](./initial.png){:style="width='100%' height='auto'"}
+
+Below, a brief explanation of each configuration option:
+
+#### Enabled
+
+This determines whether or not Federation is enabled on this server.
+
+#### Status
+
+This is the current Federation status, which can be one of the following:
+
+- **Could not enable, settings are not fully set :** Make sure all the options are correctly filled and saved;
+- **Booting... :** federation is initializing;
+- _[HUB Only]_ **Registering with Hub... :** federation is trying to register with the Hub;
+- _[HUB Only]_ **Disabled, could not register with Hub :** federation could not register with the Hub;
+- **Disabled :** federation is disabled
+- **Running :** federation is running and ready to communicate with other servers;
+
+#### Domain
+
+Fill this option with your domain name, usually where the Rocket.Chat server is hosted.
+
+#### Public Key
+
+All Federation communications are double encrypted, which means messages cannot be delivered by mistake to a wrong peer. This is your public key, which is specially generated for Rocket.Chat and by Rocket.Chat, which will be shared with peers you are communicating with.
```

Comment:
Is this totally true?  do we mean that it cannot be decrypted even if it is misdelivered?  Or do we decrypt verify and then refuse to accept that message if it is not for us?

Reply:
The encryption process encrypts the message using the Server A private key, and the Server B public key.

When a server receives the message, it uses the Server B private key and the Server A public key to decrypt it.

So, let's say the message ended up on Server C: this server won't have Server B's private key to decrypt it, so the message can't be read at all.

Reply:
Can we put that into words here in the doc?  Almost exactly as is would be good.  I think this would help others understand the encryption and how its secure also